cmd/teleproxy/teleproxy_test.go

```diff
@@ -123,3 +136,91 @@ func TestAlreadyRunning(t *testing.T) {
 		}
 	})
 }
+
+const HupConfig = "/tmp/teleproxy_test_hup_cluster.yaml"
```

Comment:
NB: Unlike the others I commented on, this SHOULD NOT obey `$TMPDIR`.  Perhaps add a comment saying that, so no one is tempted to come along and change it to use `os.TempDir()`?